### DISCUSSION TITLE
Options not recognized on angular

### DISCUSSION THREAD

**Discussion by AdamOndrejkovic** [matteobruni/tsparticles#2339] [Q&A] [answered]
2   <ng-particles  id="tsparticles" [options]="options"></ng-particles>
                                     ~~~~~~~

  src/app/todos/todos-overview/todos-overview.component.ts:7:16
    7   templateUrl: './todos-overview.component.html',
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Error occurs in the template of component TodosOverviewComponent.
    
I have followed the instructions from the documentation but for some reason, it doesn't recognize the options. Does someone know why this may be and what is the solution for it? Thank you.

**Comment by matteobruni** [comment, marked as answer]
Can you add your options in the question or a sample with the error?

**Reply by AdamOndrejkovic** [reply]
I've used one of the samples from demos as an option in my project. That I have stored in options variable. I can share my project on GitHub where you could see how I've done it so it may be easier to understand where the issue occurs.

**Reply by matteobruni** [reply]
Yeah, sure.

Have you checked the demo app in the repository? It works fine and maybe it could help you identify the issue.

https://github.com/matteobruni/tsparticles/tree/main/demo/angular/src

**Reply by AdamOndrejkovic** [reply]
I have checked your demo and tried to follow the steps.

I will attach the repository. The file that I am using particles in is under src -> app -> todos -> todos-overview -> in todos-overview.component.html the ng-particles is commented out. When I run it as mentioned it say the option is not recognized. My option is in the todos-overview.component.ts file 
https://github.com/AdamOndrejkovic/petshop-frontend

**Reply by matteobruni** [reply]
I've checked your repository and there are some error in the options object. You can see them casting the options object directly to `ISourceOptions` interface from `tsparticles` like this:

``` typescript
options: ISourceOptions = {
```
